src/main/java/com/fasterxml/jackson/databind/JsonNode.java

```diff
@@ -1071,6 +1071,8 @@ public boolean hasNonNull(int index) {
      * this node is a JSON Array or Object node. In case of Object node,
      * field names (keys) are not included, only values.
      * For other types of nodes, returns empty iterator.
+     * <p>
+     * Note: In Jackson 3, this method will be renamed to {@code values()}.
```

Comment:
Either this.... or do both of below

1. `@Deprecated` elements() 
2. Add new `values()` method in 2.19 version

Reply:
I'll merge this, would accept PR for (2) too -- let's wait bit longer until doing (1) (like 2.20).

Reply:
Actually, one note: I will file another issue for changing return type of `values()` to `Collection<JsonNode>`, to align with other accessors (`properties()`, `propertyNames()`) -- easier to use than `Iterator`.

Reply:
@JooHyukKim Ok, I decided to revisit #4869, see changed description. So part is done for 2.19, rest for 3.0.

Reply:
Makes sense thanks!